parity/run.rs

```diff
@@ -745,6 +745,7 @@ fn execute_impl<Cr, Rr>(cmd: RunCmd, logger: Arc<RotatingLogger>, on_client_rq:
 		gas_price_percentile: cmd.gas_price_percentile,
 		poll_lifetime: cmd.poll_lifetime,
 		allow_missing_blocks: cmd.allow_missing_blocks,
+		no_ancient_blocks: cmd.download_old_blocks,
```

Comment:
shouldn't this be the other way around?
```suggestion
		no_ancient_blocks: !cmd.download_old_blocks,
```

Reply:
Was going to suggest the same :)

Reply:
actually you're right, good catch. :sweat_smile: